### PR TITLE
Rendering user input when template invocation is provided

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

File: test/unit/input_template_renderer_test.rb [hunk 1]
```diff
@@ -57,13 +65,17 @@
         end
 
         context 'with invocation specified' do
-          let(:result) do
-            renderer.invocation = nil # TODO
-            renderer.render
+
+          before do
+            @input_value = FactoryGirl.create(:template_invocation_input_value,
```
Comment (ares) [inline test/unit/input_template_renderer_test.rb]: where is `@input_value` used?

File: test/unit/input_template_renderer_test.rb [hunk 2]
```diff
@@ -57,13 +65,17 @@
...
+                                              :template_invocation =>  template_invocation,
+                                              :template_input      => template.template_inputs.first,
```
Comment (ares) [inline test/unit/input_template_renderer_test.rb]: please don't indent hashes, this is not puppet \o/ (both lines above have extra spaces)

**Comment by ares** [comment]
Two nits, then good to go.

File: test/unit/input_template_renderer_test.rb (hunk 2 above)
Comment (iNecas) [inline test/unit/input_template_renderer_test.rb]: Thank God it's not puppet! :) Both updated, will merge after travis

**Pull request merged**